skada/datasets/_samples_generator.py

```diff
@@ -25,7 +25,10 @@ def _generate_unif_circle(n_samples, rng):
     return x
 
 
-def _generate_data_2d_classif(n_samples, rng, label='binary'):
+def _generate_data_2d_classif(
+        n_samples, rng, mu_regression=np.array([0, 0]),
+        sigma_regression=np.array([[1, 0], [0, 1]]),
```

Comment:
never put mutable variable (such as lists or numpy arrays) as default parameter of a function. Use Noe and set the value to default is None in the function.